docs/supported-libraries.md

```diff
@@ -17,111 +17,134 @@ or [contributing](../CONTRIBUTING.md).
+| [Armeria](https://armeria.dev)                                                                                                              | 1.3+                          | [opentelemetry-armeria-1.3](../instrumentation/armeria-1.3/library)                                                                                                                                                                                                                                                                                                            | TODO                                                                                   |
+| [AsyncHttpClient](https://github.com/AsyncHttpClient/async-http-client)                                                                     | 1.9+                          | N/A                                                                                                                                                                                                                                                                                                                                                                            | TODO                                                                                   |
+| [AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/java-handler.html)                                                                | 1.0+                          | [opentelemetry-aws-lambda-core-1.0](../instrumentation/aws-lambda/aws-lambda-core-1.0/library), [opentelemetry-aws-lambda-events-2.2](../instrumentation/aws-lambda/aws-lambda-events-2.2/library)                                                                                                                                                                             | TODO                                                                                   |
+| [AWS SDK](https://aws.amazon.com/sdk-for-java/)                                                                                             | 1.11.x and 2.2.0+             | [opentelemetry-aws-sdk-1.11](../instrumentation/aws-sdk/aws-sdk-1.11/library), [opentelemetry-aws-sdk-1.11-autoconfigure](../instrumentation/aws-sdk/aws-sdk-1.11/library-autoconfigure), [opentelemetry-aws-sdk-2.2](../instrumentation/aws-sdk/aws-sdk-2.2/library), [opentelemetry-aws-sdk-2.2-autoconfigure](../instrumentation/aws-sdk/aws-sdk-2.2/library-autoconfigure) | TODO                                                                                   |
```

Comment:
Including all the links inline makes editing this difficult. Might be a good idea to extract them out like I've done with with the semantic conventions (i.e. `[HTTP Server Spans]`). Normally I like to extract links like that when they occur more than once, but can also improve readability in this case.

Reply:
no problem, maybe also `<br>` between the links for rendering?